src/coreclr/gc/gc.cpp

```diff
@@ -2253,8 +2253,15 @@ bool        affinity_config_specified_p = false;
 #ifdef USE_REGIONS
 region_allocator global_region_allocator;
 uint8_t*(*initial_regions)[total_generation_count][2] = nullptr;
+
+uint8_t* bookkeeping_data = nullptr;
+uint8_t* bookkeeping_data_committed = nullptr;
+size_t card_table_element_sizes[total_bookkeeping_elements];
```

Comment:
I would put these as `PER_HEAP_ISOLATED` members of `gc_heap` as these are internal to the GC heap.